src/contracts/core/AllocationManager.sol

```diff
@@ -446,7 +448,9 @@ contract AllocationManager is
     function _isOperatorSlashable(address operator, OperatorSet memory operatorSet) internal view returns (bool) {
         RegistrationStatus memory status = registrationStatus[operator][operatorSet.key()];
 
-        return status.registered || block.number < status.slashableUntil;
+        // slashableUntil returns the last block the operator is slashable in so we check for
+        // than or equal to
```

Comment:
comment typo, "less than or equal to"

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/965/commits/4d27ea0980b1670d19d3f6fbea6f03972dfced12